packages/devtools_app/lib/src/screens/deep_link_validation/validation_details_view.dart

```diff
@@ -651,6 +651,11 @@ class _PathCheckTable extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final linkData = controller.selectedLink.value!;
```

Comment:
should we be null-asserting here? Or in other words are we confident the selected link will be non-null at this point?

Reply:
yes, we are confident a link is selected before showing the validation result.